### PR TITLE
DI extenstion convention /3

### LINKED ISSUE #22862: "Place extension methods in the Microsoft.Extensions.DependencyInjection namespace to encapsulate groups of service registrations." ??
Why would we put any of our code in a Microsoft namespace?  What is the value in doing that?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0eb4d762-3910-2c57-8741-3306c15537b6
* Version Independent ID: ba2254a0-ea8e-9b28-e94b-edd0fc58c043
* Content: [Configuration in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-3.1#combining-service-collection)
* Content Source: [aspnetcore/fundamentals/configuration/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/configuration/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #22862

@serpent5  please review

### CODE REVIEW

File: aspnetcore/includes/combine-di.md [hunk 1]
```diff
@@ -15,4 +15,7 @@ The remaining services are registered in a similar class. The following `Configu
...
+**_Note:_** Each `services.Add{GROUP_NAME}` extension method adds and potentially configures services. For example, <xref:Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions.AddControllersWithViews%2A> adds the services MVC controllers with views require, and <xref:Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions.AddRazorPages%2A> adds the services Razor Pages requires. We recommended that apps follow the naming convention of creating extension methods in the <xref:Microsoft.Extensions.DependencyInjection?displayProperty=fullName> namespace. Creating extension methods in the `<xref:`Microsoft.Extensions.DependencyInjection` namespace:
+
+* Encapsulates groups of service registrations.
+* Provides convient intellisense access to the service.
```
Comment (Rick-Anderson) [inline aspnetcore/includes/combine-di.md:21]:
```suggestion
* Provides convenient intellisense access to the service.
```

File: aspnetcore/includes/combine-di.md [hunk 2]
```diff
@@ -15,4 +15,7 @@ The remaining services are registered in a similar class. The following `Configu
...
+**_Note:_** Each `services.Add{GROUP_NAME}` extension method adds and potentially configures services. For example, <xref:Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions.AddControllersWithViews%2A> adds the services MVC controllers with views require, and <xref:Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions.AddRazorPages%2A> adds the services Razor Pages requires. We recommended that apps follow the naming convention of creating extension methods in the <xref:Microsoft.Extensions.DependencyInjection?displayProperty=fullName> namespace. Creating extension methods in the `<xref:`Microsoft.Extensions.DependencyInjection` namespace:
+
+* Encapsulates groups of service registrations.
+* Provides convenient intellisense access to the service.
```
Comment (guardrex) [inline aspnetcore/includes/combine-di.md:21]:
```suggestion
* Provides convenient IntelliSense access to the service.
```

File: aspnetcore/includes/combine-di.md [hunk 3]
```diff
@@ -15,4 +15,7 @@ The remaining services are registered in a similar class. The following `Configu
...
+**_Note:_** Each `services.Add{GROUP_NAME}` extension method adds and potentially configures services. For example, <xref:Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions.AddControllersWithViews%2A> adds the services MVC controllers with views require, and <xref:Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions.AddRazorPages%2A> adds the services Razor Pages requires. We recommended that apps follow the naming convention of creating extension methods in the <xref:Microsoft.Extensions.DependencyInjection?displayProperty=fullName> namespace. Creating extension methods in the `<xref:`Microsoft.Extensions.DependencyInjection` namespace:
```
Comment (guardrex) [inline aspnetcore/includes/combine-di.md:18]:
```suggestion
**_Note:_** Each `services.Add{GROUP_NAME}` extension method adds and potentially configures services. For example, <xref:Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions.AddControllersWithViews%2A> adds the services MVC controllers with views require, and <xref:Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions.AddRazorPages%2A> adds the services Razor Pages requires. We recommended that apps follow the naming convention of creating extension methods in the <xref:Microsoft.Extensions.DependencyInjection?displayProperty=fullName> namespace. Creating extension methods in the `Microsoft.Extensions.DependencyInjection` namespace:
```

**Review by serpent5 — APPROVED** [review]
👍

**Comment by Rick-Anderson** [comment]
@serpent5 much appreciated.

**Pull request merged**